CHANGELOG.md

```diff
@@ -4,6 +4,51 @@ All notable changes to Zebra will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org).
 
+## [Zebra 1.0.0-alpha.13](https://github.com/ZcashFoundation/zebra/releases/tag/v1.0.0-alpha.13) - TODO: Update with release date before merging
+
```

Comment:
We usually do a release summary here - this is a copy of the last one with placeholder text:

```
Zebra's latest alpha continues our work on NU5, including Orchard and Transaction V5.
(Mention any important new features or fixes here.)
```

Reply:
Done, but I'm not sure if I selected the most important parts :thinking:

Reply:
Seems fine to me!